get.go

```diff
@@ -0,0 +1,52 @@
+package helmclient
+
+import (
+	"context"
+
+	"github.com/giantswarm/microerror"
+	"github.com/prometheus/client_golang/prometheus"
+	"helm.sh/helm/v3/pkg/action"
+	"helm.sh/helm/v3/pkg/release"
+)
+
+// GetReleaseContent gets the current status of the Helm Release including any
+// values provided when the chart was installed. The releaseName is the name
+// of the Helm Release that is set when the Helm Chart is installed.
+func (c *Client) GetReleaseContent(ctx context.Context, releaseName, namespace string) (*ReleaseContent, error) {
```

Comment:
Method signature needs to change to take in the namespace as well as the release name.

This is so the helm kube client is scoped to the namespace. Before we just configured the tiller namespace.